docs/reference/commands/build.rst

```diff
@@ -0,0 +1,51 @@
+
+.. _ref_commands_build:
+
+build
+=====
+
+Build artifacts defined for a part. If part names are specified only
+those parts will be built, otherwise all parts will be built.
```

Comment:
I'm thinking if we maybe should mention a bit about how the parts for an imagecraft image look like here. Maybe we should think about it and return to it later though.

Reply:
See previous comments. I agree with you explaining what is going on is valuable but it should probably be in the `explanation` section.